src/EditorFeatures/CSharpTest/Squiggles/ErrorSquiggleProducerTests.cs

```diff
@@ -310,7 +310,7 @@ class Test
                     TestDiagnosticTagProducer<DiagnosticsSquiggleTaggerProvider, IErrorTag>.CreateDiagnosticData(document, new TextSpan(0, 0)),
                     TestDiagnosticTagProducer<DiagnosticsSquiggleTaggerProvider, IErrorTag>.CreateDiagnosticData(document, new TextSpan(0, 1))));
 
-            var spans = await TestDiagnosticTagProducer<DiagnosticsSquiggleTaggerProvider, IErrorTag>.GetErrorsFromUpdateSource(workspace, updateArgs);
+            var spans = await TestDiagnosticTagProducer<DiagnosticsSquiggleTaggerProvider, IErrorTag>.GetErrorsFromUpdateSource(workspace, updateArgs, DiagnosticKind.CompilerSyntax);
```

Comment:
@CyrusNajmabadi I wasn't sure if there is value in making this test a `Theory` instead of `Fact` and testing for each individual `DiagnosticKind`. Feel free to do so if you think it adds value.